### DISCUSSION TITLE
how to make a reset pricescale default when double click on pricescale range?

### DISCUSSION THREAD

**Discussion by huypham1411** [tradingview/lightweight-charts#1667] [Q&A] [answered]
Hi, is there any way to reset the price scale default when double-click inside the price scale zone like Tradingview? or detect if the mouse is inside the price scale zone

**Comment by edew** [comment, marked as answer]
Yes and it's enabled by default.

https://tradingview.github.io/lightweight-charts/docs/api/interfaces/HandleScaleOptions#axisdoubleclickreset

![lwc_price_scale_doubleclick](https://github.com/user-attachments/assets/39775672-fb59-48b6-aa5f-726a9293e0cf)

**Reply by huypham1411** [reply]
I don't know why but my pricescale doesn't behave like default, is there any example sir? 
I created a quick codesandbox using code from react guide doc [link](https://codesandbox.io/p/sandbox/lightweight-chart-multi-screen-test-cz427c?file=%2Fsrc%2FChart.jsx)
and also is there anyway to know if the mouse in in the pricescale zone? Thanks a lot
